mypy/util.py

```diff
@@ -721,3 +722,31 @@ def is_stub_package_file(file: str) -> bool:
         return False
     return any(component.endswith('-stubs')
                for component in os.path.abspath(file).split(os.sep))
+
+
+def get_sitepkgs_prefix(path: pathlib.Path) -> Optional[pathlib.Path]:
+    if path.name != "site-packages":
+        return None
+    # it is either <prefix>/[Ll]ib/site-packages or <prefix>/lib/pythonX.Y/site-packages
+    # so max 2 levels up
+    prefix = path.parent
+    for _ in range(2):
+        if prefix.name.lower() == "lib":
+            return prefix.parent
+        prefix = prefix.parent
+    return None
+
+
+def is_default_sitepkgs(path: pathlib.Path) -> bool:
+    return path.resolve() in (pathlib.Path(_).resolve() for _ in sitepkgs.getsitepackages())
```

Comment:
Nit: I would use a name instead of an underscore, perhaps use `sitepkg_dir` instead.